### PR TITLE
Fix NaCl build: tar was called incorrectly.

### PR DESCRIPTION
**Pull request by iphydf**
tar -jxf expects a file.

---
This change is [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/410)

### CODE REVIEW

File: other/travis/autotools-linux-install [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 # Also install vanilla NaCl
 [ -f $CACHE_DIR/lib/amd64/libnacl.a ] || {
-  curl https://hyperelliptic.org/nacl/nacl-20110221.tar.bz2 | tar -jxf
+  curl https://hyperelliptic.org/nacl/nacl-20110221.tar.bz2 | tar jx
```
Comment (zetok) [inline other/travis/autotools-linux-install]: Specifying archive format is not necessary when calling `tar`.

**Comment by iphydf** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[other/travis/autotools-linux-install, line 7 at r1](https://reviewable.io:443/reviews/toktok/c-toxcore/410#-K_ywaD_rTUCMtIcS2r--r1-7:-K_z1JhKBxYlNjeAFGKs:bbdv9ba) ([raw file](https://github.com/toktok/c-toxcore/blob/20be9a8c5b3cf41fc766123bcc6e1efbe9b2b86e/other/travis/autotools-linux-install#L7)):*
<details><summary><i>Previously, zetok wrote…</i></summary><blockquote>

Specifying archive format is not necessary when calling `tar`.
</blockquote></details>

It is for pipes. It can't infer the format without a filename.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/410)*

**Comment by zetok** [comment]
:lgtm_strong:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/410#-:-K_z6Q6yEtSk8TPibmkj:b-jbsf8z)*

**Comment by nurupo** [comment]
:lgtm_strong:

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/410#-:-KaKBJw_JFvx-d1yNSvS:b-jbsf8z)*

**Pull request merged**